README.rst

```diff
@@ -294,13 +294,13 @@ directory of your choice::
     cd synapse
 
 Synapse has a number of external dependencies. We maintain a fixed development
-environment using [poetry](https://python-poetry.org/). First, install poetry. We recommend
+environment using `Poetry <https://python-poetry.org/>`_. First, install poetry. We recommend::
```

Comment:
Arg. Is there a ReST linter that might have caught this? Maybe https://pypi.org/project/restructuredtext-lint/

Reply:
FWIW, my training/tradition is to treat displayed items as a noun in the flow of a sentence, including punctuation if ending the sentence. Though this doesn't work brilliantly for source code.

Reply:
> Arg. Is there a ReST linter that might have caught this? Maybe https://pypi.org/project/restructuredtext-lint/

I've never looked into this, tbh. Given the README is the only RST thing we have nowadays, I'm not sure it's worth much linting effort, though I'll not statnd in the way.

Reply:
> FWIW, my training/tradition is to treat displayed items as a noun in the flow of a sentence, including punctuation if ending the sentence. Though this doesn't work brilliantly for source code.

Afraid I'm not really sure what you're referring to here. The capitalisation of "Poetry"? The `::` ?

Reply:
> I've never looked into this, tbh. Given the README is the only RST thing we have nowadays, I'm not sure it's worth much linting effort, though I'll not statnd in the way.

Further, it's not obvious what there is to "catch": it was valid RST, it just didn't do what you wanted it to do. markdown-style links are a common error that we could look for explicitly, I suppose, but I'm not sure it's reasonable to expect any generic RST linter to do so.

Reply:
> > FWIW, my training/tradition is to treat displayed items as a noun in the flow of a sentence, including punctuation if ending the sentence. Though this doesn't work brilliantly for source code.
> 
> Afraid I'm not really sure what you're referring to here. The capitalisation of "Poetry"? The `::` ?

Sorry. I meant the omission of the trailing `::`. The intent was that the following is all one sentence:

> We recommend `<block of shell>`, as described here.

But that's a mionr style thing that isn't worth squabbling over.

Reply:
I want a program that knows all of the mistakes that I make in advance and catches them. AKA, the moon on a stick.

Reply:
> Sorry. I meant the omission of the trailing `::`. The intent was that the following is all one sentence:

Oh right. `::` in RST has special syntactic meaning (https://docutils.sourceforge.io/docs/user/rst/quickref.html#literal-blocks). Not sure if there is a way to get that effect without also getting a `:` in the output.